docs/conf.py

```diff
@@ -25,10 +30,9 @@
     sys.exit(1)
 
 # Get configuration information from setup.cfg
-conf = ConfigParser()
-conf.read([os.path.join(os.path.dirname(__file__), '..', 'setup.cfg')])
-setup_cfg = dict(conf.items('metadata'))
-
+with open(Path(__file__).parent.parent / 'pyproject.toml', 'rb') as configuration_file:
+    configuration = tomllib.load(configuration_file)
+metadata = configuration['metadata']
```

Comment:
```suggestion
metadata = configuration['project']
```

This should fix the RTD build.